src/applications/vaos/services/appointment/index.js

```diff
@@ -140,3 +143,42 @@ export function getVAAppointmentLocationId(appointment) {
 
   return null;
 }
+
+/**
+ * Returns the location name of a VA appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns The location name where the VA appointment is located
+ */
+export function getVAAppointmentLocationName(appointment) {
+  return appointment.participant?.find(p =>
+    p.actor.reference?.startsWith('Location'),
+  )?.actor?.display;
+}
+
+/**
+ * Returns the patient phone number of a VA appointment
+ *
+ * @export
+ * @param {Object} appointment A FHIR appointment resource
+ * @returns The patient phone number where the VA appointment is located
+ */
+export function getPatientPhone(appointment) {
+  return appointment.participant
+    .find(p => p?.actor?.reference.includes('Patient'))
+    ?.actor?.telecom?.find(t => t?.system === 'phone')?.value;
```

Comment:
It could be null if someone was to try to pass a confirmed appointment to this function.  Do you think we should return `null` if it's undefined? Or possibly just an empty string?

Reply:
I think what you're returning is fine. I was trying to figure out if we need the `t?.system` vs `t.system`. This would only matter if there were null or undefined items in the `telecom` array, which seems like it would be an error.

Reply:
t wouldn't ever be null, but participant could be null.  Moving the optional chaining to be on participant before `.find()`.